src/app/redux/TransactionSaga.js

```diff
@@ -159,6 +168,10 @@ function* broadcastOperation({payload:
     }
 }
 
+function hasPrivateKeys(payload) {
```

Comment:
because of the optionality of the P, this will trigger for literally any 50-character alphanumeric string that starts with a decimal 5. is there really no other way to validate a private key?

Reply:
Not literally any alphanumeric string because the keys are base58 :) but yeah, there will be false positives. The keys have a checksum, I can add a check for that as well

Reply:
Strictly speaking, Base58 compliant string so unlikely for false positives on such long scripts but we can also perform [WIF checksum checking](https://en.bitcoin.it/wiki/Wallet_import_format)

Reply:
I dont think checksum checking is necessary. What if they mistranscribe it? We still want to block secret keys and passwords that are slightly off. I am fine with this solution.

Reply:
It's not too uncommon for our imagehoster hashes to create false positives, I think we should add the check, and I already wrote it so... :)

Reply:
Done! (@goldibex please try to put review comments on a line that would be changed so they get invalidated)